### PR TITLE
Compatibility with Sulvo.com ads

### PR DESCRIPTION
**Pull request by afuggini**

### CODE REVIEW

**Comment by afuggini** [comment]
It is signed as Sulvo! @sulvoteam

**Comment by zhouyx** [comment]
@afuggini  Thanks for the Pull Request! A few things at first.

- Please follow our [Developer Guildlines for a Pull Request](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request) to change all related files.
- Please consider implementing the render-start and no-content-available APIs (see [Available APIs](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis)), which helps AMP to provide user a much better ad loading experience.

Please let me know if there's any question.

**Comment by afuggini** [comment]
@zhouyx Thank you for your feedback. I resolved the issues pointed by CI. We were basically in conflict with `spotx` code, I am not sure why.
I also implemented `render-start` and would rather `no-content-available` for a future version, unless this is mandatory.

Looking forward to your feedback!
Thanks.

File: ads/sulvo/sulvo.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```
Comment (zhouyx) [inline ads/sulvo/sulvo.js]: 2017

File: ads/sulvo/sulvo.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+import {validateData, loadScript} from '../../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function sulvo(global, data) {
+
+  validateData(data, [
```
Comment (zhouyx) [inline ads/sulvo/sulvo.js]: nit: would prefer ['adClient', 'adDomain', 'adId'] ...

File: ads/sulvo/sulvo.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    'adClient',
+    'adCustom',
+    'adDomain',
+    'adId',
+  ]);
+
+  // Create Sulvo body tag
+  const c = global.document.createElement('div');
+  const slot = global.document.getElementById('c');
+  const url = `https://surgeprice.com/display/async/${data.adClient}/${data.adDomain}/ariel.js`;
```
Comment (zhouyx) [inline ads/sulvo/sulvo.js]: nit: 80 chars limit

Comment (zhouyx) [inline ads/sulvo/sulvo.js]: nit: use `encodeURIComponent` please

File: examples/sulvo.amp.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```
Comment (zhouyx) [inline examples/sulvo.amp.html]: please delete the file

**Comment by zhouyx** [comment]
@afuggini Any update here?

**Comment by afuggini** [comment]
Hi @zhouyx, we are reworking our ad server. I will update the pull request once it is ready. It might take a couple months until we are ready. Thanks for checking in.

**Comment by zhouyx** [comment]
Close PR. @afuggini Feel free to reopen it later.

**Pull request closed without merging**